src/subcommands/deploy.ts

```diff
@@ -28,15 +29,18 @@ USAGE:
     deployctl deploy [OPTIONS] <ENTRYPOINT>
```

Comment:
```suggestion
    deployctl deploy [OPTIONS] [<ENTRYPOINT>]
```

Reply:
Let's see first where we land with the flag vs positional param discussion